store.js

```diff
@@ -0,0 +1,283 @@
+const { pack, unpack }      = require('msgpackr')
+const { mkdirSync }         = require('fs')
+const { Cursor, Env }       = require('node-gyp-build')(__dirname)
+const { AsyncLocalStorage } = require('async_hooks')
+
+function asBinary(buffer) {
+  return {
+    ['\x10binary-data\x02']: buffer,
+  }
+}
```

Comment:
```suggestion
const BINARY_DATA_KEY = '\x10binary-data\x02'

function asBinary(buffer) {
  return {
    [BINARY_DATA_KEY]: buffer,
  }
}
```

plus other usage of this constant

Reply:
Thanks Sam, done.